airflow/utils/log/json_formatter.py

```diff
@@ -21,12 +21,18 @@
 import json
 import logging
 
+import pendulum
```

Comment:
This is part of the airflow core.  This change changes elasticsearch provider so there are two problems: 

1) What happens if the new provider is installed with old airflow? Do we have some dependency here on those modifications? I guess yes because otherwise you would not not change it. Can you please make it standalone in the way that it will not require changes in airlfow core?

2) Secondly (related)- what's the impact of this change on the rest of Airlfow? How formatTime addition will impact any other place  that uses JSONFormatter? Did you try to assess the impact?

Reply:
@potiuk I made a separate formatter for ES. So now it won't break compatibility